apache-hadoop-mixin/dashboards/hadoop-datanode-overview.libsonnet

```diff
@@ -338,10 +340,22 @@ local datanodeLogsPanel = {
               allValues='.+',
               sort=1
             ),
+            template.new(
+              'cluster',
+              promDatasource,
+              'label_values(hadoop_datanode_ramdiskblocksevictedwithoutread{%(hadoopSelector)s}, cluster)' % $._config,
```

Comment:
I don't know if you've been doing this pattern before and it works fine, but does it make sense to have `cluster` in the filter for the query for itself (the variable `cluster`)?

Reply:
This is our established pattern for previous k8s mixins, so unless something has changed I think this is good 👍